experiment/simulation/css/main.css

```diff
@@ -1 +1,228 @@
-/* You CSS goes in here */
\ No newline at end of file
+.main{
+    display: flex; 
+    justify-content: center; 
+    align-items: center;
+}
+
+.title {
+    text-align: center;
+    margin-top: 5px;
+    font-size: 2.5vw;
+    z-index: -1;
+}
+
+.nav-link ,.navbar-nav,.navbar ,.navbar-custom{
+    padding-left: 15px;
+    padding-right: 15px;
+    background-color: rgb(21,100,156) ;
+}
+.nav-link{
+    color:rgb(169,169,169);
+}
+
+.activeLink{
+    color: rgb(151, 220, 61);
+}
+.navbar-nav>li>a:focus,.navbar-nav>li>a:hover{
+    color: rgb(151, 210, 61);
+    background-color: rgb(21,100,156) ;
+}
+
+.procedureMessage {
+    text-align: center;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 1.4vw;
+}
+
+.instructionsBox{
+    margin-top: 10px;
+    padding: 0px 20px;
+    border-radius: 20px;
+    border: 3px solid rgb(151, 210, 61) ;
+}
+
+.molecularView, .dropping-objects{
+    position: relative;
+}
+
+
+.micelle-head{
+    margin-top: 70px;   
+}
+
+.micelle-tail{
+    margin-top: 90px;
+}
+
+.micelle-head:hover, .micelle-tail:hover{
+    transform: scale(1.2);
+    transition: all 1s ease;
+}
+
+.img-micelle{
+    padding-top: 60px;
+}
+
+.micelle-information{
+    font-size: 2vw;
+}
+
+.bench {
+    z-index: -1;
+    position: absolute;
+    align-self: center; 
+    object-fit: cover;
+    top: 10vw;
+    width: 78vw;
+    left: 10vw;
+}
+
+.spoon1, .spoon2, .spoon3{
+    position: absolute;
+    align-self: center;
+    object-fit: cover;
+    top: 1vw;
+    right: 10vw;
+    width: 15vw;
+}
+
+.detergent1,.detergent2,.detergent3{
+    position: absolute;
+    align-self: center;
+    object-fit: cover;
+    top: 3vw;
+    right: 19vw;
+    width: 6vw;
+}
+
+.spoon2, .spoon3, .detergent2, .detergent3{
```

Comment:
Create a separate class for opacity named as is-opaque or is-transparent or is-not-opaque. That way you'd be able to reuse that class as needed. This way you are just replacing the id based styling with classnames. It does not achieve the purpose of using class based styling.

Reply:
Keeping opacity in the same classes will help in anime animation.